spaceR/spaceR/Models/Launchpads.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Launchpads.swift
+//  SpaceR
+//
+//  Created by Kieran Woodrow on 2022/03/16.
+//
+
+import Foundation
+
+struct Launchpads: Codable {
+    
+    let launchID: String?
+    let lauchpadName: String?
+    let launchpadStatus: String?
+    let location: String?
+    let region: String?
+    let timezone: String?
+    let latitude: Double?
```

Comment:
Just to be sure please make sure that there are not just decoding to nil 😃 also another note if you plan to use all this information great 😄 but if not then you don't need to decode the entire model.